content/en/docs/concepts/secure-deployment.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Securely deploying Notation"
+description: "Best practices for securely deploying Notation"
+type: docs
+weight: 2
+---
+
+A deployment of Notation, specifically the `notation` CLI, is used for signing and verifying container images. The `notation` CLI is usually deployed on a development computer or in a CI/CD pipeline, such as GitHub Actions.
+
+A deployment has several sensitive components:
+
+- [Trust policies]({{< ref "/docs/how-to/manage-trust-policy" >}})
+- [Trust stores and certificates]({{< ref "/docs/concepts/directory-structure" >}})
+- Plugins
+
+It is strongly recommended that you follow the best practices in this document to secure your deployment of Notation. Failure to secure the filesystem on development computers and hosts with `notation` CLI deployments can result in malicious plugins being installed, trust policies being modified to allow verification compromised or untrusted artifacts, and trust stores being modified to have untrusted or compromised certificates.
```

Comment:
Should we callout Notation works in shared responsibility model. Users are responsible for securing the host on which notation is deployed ?

Reply:
@priteshbandi  i feel like this line states this fact without calling it "shared responsibility". Do you feel like we need to state this more explicitly?

Reply:
Yeah we should be explicit about shared responsibility.

Reply:
@zr-msft Please check this comment from Pritesh on threat modeling PR, https://github.com/notaryproject/notaryproject/pull/242#discussion_r1223622951, it will help you to understand shared responsibility.